apps/web/screens/Grants/Grant/GrantSidebar.tsx

```
@@ -0,0 +1,121 @@
+    [grant, formatMessage, linkResolver],
+  )
+
+  const filesPanelData = useMemo(
+    () =>
+      grant.files
+        ?.map((f) => {
+          if (!f.url) {
+            return null
+          }
+          return (
+            <LinkV2 href={f.url} underlineVisibility="hover">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)